include/util/graph_loader.hpp

```diff
@@ -32,27 +33,17 @@ namespace util
  * The since the restrictions reference nodes using their external node id,
  * we need to renumber it to the new internal id.
 */
-inline unsigned loadRestrictionsFromFile(std::istream &input_stream,
+inline unsigned loadRestrictionsFromFile(std::string &filename,
```

Comment:
if we pass in a filename, it should be `const`. Any reason for the switch to the filename?
It's less flexible, since we cannot store multiple items in a single file anymore. At some point, we probably want to get rid of all these 1000 files and have many items combined in a file. At that point, we would have to switch back to a stream again.